.github/labeler.yml

```diff
@@ -0,0 +1,17 @@
+schedule:
+- any: ['src/components/SchedulePage/**/*', 'src/pages/schedule.js']
+
+homepage:
+- any: ['src/components/HomePage/**/*', 'src/pages/index.js']
+
+prizes:
+- any: ['src/components/PrizePage/**/*', 'src/pages/prizes.js']
+
+page:
+- any: ['src/components/MenuBar/**/*', 'src/components/Footer/**/*', 'src/components/layout.js']
+
+gallery:
+- any: ['src/components/GalleryPage/**/*', 'src/pages/gallery.js']
+
+infra:
+- any: ['.github/**/*', '!.github/FUNDING.yml']
```

Comment:
Another thing is that I think you meant to say "anything in `.github`, except `FUNDING.yml`". If this is the case, then you should do

```yaml
infra:
- any: ['.github/**/*']
  all: ['!.github/FUNDING.yml']
```

See https://github.com/actions/labeler#common-examples

Reply:
Wait, wouldn't that one be: "any change to `.github/**/*` files as long as the `.github/FUNDING.yml` hasn't changed"

Reply:
Ah, I think you're right. The status quo looks good then.